bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -56,12 +57,49 @@ spec:
               exit 0
           }
           trap quit SIGTERM
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
-          ovs-appctl vlog/set "file:${OVS_LOG_LEVEL}"
-          /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=6081 enable-protocol
+          chroot /host /bin/bash <<EOF
```

Comment:
OK and looking at those denials...I think probably what we need to do is write this script out to the host at e.g. `/run/bin/openshift-cno-ovs` and then use:
`chroot /host systemd-run --unit=openshift-cno-ovs /run/bin/openshift-cno-ovs` or so to fully escape the container and execute with the host `unconfined_t` context.

It may also be possible to use `unconfined_t` instead of `spc_t`.